host/libhackrf/src/hackrf.h

```diff
@@ -66,6 +66,7 @@ enum hackrf_board_id {
 	BOARD_ID_JELLYBEAN  = 0,
 	BOARD_ID_JAWBREAKER = 1,
 	BOARD_ID_HACKRF_ONE = 2,
+	BOARD_ID_RAD1O = 3,
```

Comment:
Does the rad1o firmware report a board ID of 3?  Last time I used it, I thought it was reporting 2 (HackRF One).